### DISCUSSION TITLE
Star Chinese Movies

### DISCUSSION THREAD

**Discussion by AzeemHadzrie** [iptv-org/community#799] [Q&A]
### Channel Name

Star Chinese Movies

### Channel ID

StarChineseMovies.hk

### Website

dngtw.tw/SCM

### Notes

A Chinese Movie channel owned by Disney Networks Group Asia Pacific, I can't find any working m3u8.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19787
